packages/terra-folder-tree/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I have noticed that in that and FilePath component we have `.eslintrc.js` and `jest.config.js` on package level rather than use them from terra-framework level and then use as in older components:
`"jest": "jest --config ../../jest.config.js"`
`"lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore"`
I was wondering why we switched to that?

Reply:
Either way works for me. I am just creating a new component set up for Compact Interactive List and was wondering which approach to use, and if separate config - in what cases, hence asked.

Reply:
I tried removing the package-level `.eslintrc.js` and running eslint from the package level, and it errors due to not having an eslint config. I think this is because the root eslint config is defined in the root package.json, so it can't be resolved from the package level?

We could probably define a root-level `.eslintrc.js` to fix this; for now I'm going to leave the files as they are so that the Folder Tree package is available in main, but I'm open to changing this in a future PR.